libraries/fiff/fiff_id.h

```diff
@@ -152,20 +144,6 @@ class FIFFSHARED_EXPORT FiffId {
     fiff_int_t version;     /**< File version. */
     fiff_int_t machid[2];   /**< Unique machine ID. */
     fiffTimeRec time;       /**< Time of the ID creation. */
-
-// ### OLD STRUCT ###
-///**
-//* A file ID.
-//*
-//* These universially unique identifiers are also
-//* used to identify blocks within fthe files.
-//*/
-//typedef struct _fiffIdRec {
-//    fiff_int_t version;     /**< File version. */
-//    fiff_int_t machid[2];   /**< Unique machine ID. */
-//    fiffTimeRec time;       /**< Time of the ID creation. */
-//} fiffIdRec,*fiffId;        /**< This is the file identifier. */
-//typedef fiffIdRec fiff_id_t;
```

Comment:
I always found having these MNE C structs very useful from time to time. Any reason why you want to delete them?